deepforest/predict.py

```diff
@@ -154,27 +156,18 @@ def predict_file(model,
 
     Args:
         model: deepforest.main object
-        csv_file: path to csv file
+        trainer: a pytorch lightning trainer object
+        dataloader: pytorch dataloader object
         root_dir: directory of images. If none, uses "image_dir" in config
         nms_thresh: Non-max supression threshold, see config["nms_thresh"]
+        df: pandas dataframe with bounding boxes, label and scores for each image in the csv file
```

Comment:
This isn't provided as an argument and matches the previous description for the return value. Seems like this shouldn't be here?